articles/anomaly-detection/guides/set-anomaly-detection-preferences.md

```diff
@@ -1,34 +1,53 @@
 ---
 title: Set Anomaly Detection Preferences
-description: Learn how to set anomaly detection preferences in the Dashboard.
+description: Learn how to set anomaly detection preferences in the Dashboard for Automated Attack Protection, Brute-force Protection, and Breached Password Detection.
+toc: true
 topics:
     - security
     - anomaly-detection
     - brute-force-protection
     - breached-password-detection
+    - automatic-attack-protection
 contentType: how-to
-useCase: customize-anomaly-detection
-v2: true
+useCase: anomaly-detection
 ---
 # Set Anomaly Detection Preferences
 
-Customize the actions that occur after the triggers in the **Anomaly Detection** section on the [Dashboard](${manage_url}/#/anomaly).
+Enable, disable, and customize **Anomaly Detection** settings in the [Dashboard](${manage_url}/#/anomaly).
 
 ::: warning
 Auth0 recommends that you **do not** make changes to your anomaly detection features with the Management API.
 :::
 
 ![Anomaly Detection Dashboard](/media/articles/anomaly-detection/anomaly-detection-overview.png)
 
-## Brute-force protection preferences
+## Automated Attack Protection preferences
 
-Brute-force protection is enabled by default for all connections. For more information, see [Enable and Disable Brute-Force Protection](/anomaly-detection/guides/enable-disable-brute-force-protection).
+Automated attack protection protects against [credential stuffing attacks](/anomaly-detection/concepts/credential-stuffing). It is enabled by default for all connections. It provides a basic level of protection against certain attacks that does not add any friction to legitimate users. When such an attack is detected, it displays a CAPTCHA step in the login experience to eliminate bot and scripted traffic.
 
 ::: warning
-Auth0 strongly recommends that you **do not** set the `brute_force_protection` flag to `false` (effectively disabling brute-force protection for the connection), however if you do, you can change it back in the [Dashboard](${manage_url}/#/anomaly). 
+Auth0 strongly recommends that you **do not** disable **Automated Attack Protection**, however if you do, you can enable it in the [Dashboard](${manage_url}/#/anomaly).
 ::: 
 
-Limit the amount of signups and failed logins from a suspicious IP address. For more information, see [Brute-Force Protection Triggers and Actions](/anomaly-detection/references/brute-force-protection-triggers-actions).
+1. Click on the **Automated Attack Protection** shield.
+
+![Automated Attack Protection Shield]()
```

Comment:
Mocks here: https://www.figma.com/file/ygmwtr10OyXttwN8onPLjV/CAPTCHA-configuration?node-id=0%3A1

Can provide real UX render pending some PR's in flight.